src/store/messages/api.ts

```diff
@@ -11,3 +11,15 @@ export async function fetchMessagesByChannelId(channelId: string, lastCreatedAt?
 
   return response.body;
 }
+
+export async function sendMessagesByChannelId(
+  channelId: string,
+  message: string,
+  mentionedUser: string
+): Promise<boolean> {
+  const response = await Request.post(`${config.ZERO_API_URL}/chatChannels/${channelId}/message`)
+    .withCredentials()
+    .send({ message, mentionedUsers: [mentionedUser] });
```

Comment:
let's wrap the request object so that we don't need to provide the api url every time.
we should also not need to add `.withCredentials()`. that should just happen automatically.